rpi1wire/__init__.py

```diff
@@ -69,20 +69,16 @@ def __init__(self, sh):
             else:
                 self.cycle = 120
         except KeyError as e:
-            self.logger.critical(
-                "Plugin '{}': Inconsistent plugin (invalid metadata definition: {} not defined)".format(self.get_shortname(), e))
+            self.logger.critical(f"Plugin '{self.get_shortname()}': Inconsistent plugin (invalid metadata definition: {e} not defined)")
             self._init_complete = False
```

Comment:
Since this code will only run on a Raspi or compatible hardware it might be adviseable to check the hardware before proceeding. Maybe https://raspberrypi.stackexchange.com/questions/5100/detect-that-a-python-program-is-running-on-the-pi helps to find a solution here